### DISCUSSION TITLE
ETA on v3 release

### DISCUSSION THREAD

**Discussion by briavers** [orestbida/cookieconsent#488] [Q&A] [answered]
Is there any eta known when v3 will go out of beta?

**Comment by orestbida** [comment, marked as answer]
As of now the latest `rc.13` is stable.

Prior to the official release, I want to have a decent ['demo page'](https://playground.cookieconsent.orestbida.com/) (similar to v2's demo page) and revise the documentation as there are some outdated sections.

It could take 1 week or 6 months ...
